deploy/templates/nstemplatetiers/base/cluster.yaml

```diff
@@ -164,6 +164,22 @@ objects:
     name: ${USERNAME}-stage
   spec:
     timeoutSeconds: ${{IDLER_TIMEOUT_SECONDS}}
+- apiVersion: dbaas.redhat.com/v1alpha1
+  kind: DBaaSTenant
+  metadata:
+    annotations:
+      toolchain.dev.openshift.com/optional-resource: "yes"
+    name: "tenant-${USERNAME}-stage"
+  spec:
+    inventoryNamespace: ${USERNAME}-stage
```

Comment:
might not be needed once if we drop the `-stage` namespaces before merging this PR

Reply:
This PR has to be merged today and dropping the namespace will be done in the following days

Reply:
ok, that was just a remark in case the ns drop would happen earlier